src/Core/src/Platform/Android/Resources/values/styles.xml

```diff
@@ -66,6 +67,10 @@
       <item name="android:minHeight">0dp</item>
     </style>
 
+    <style name="MauiAlertDialogTheme" parent="ThemeOverlay.MaterialComponents.MaterialAlertDialog">
+      <item name="colorPrimary">?attr/colorOnSurface</item>
```

Comment:
This looks like it might override the default xaml style values? 

I'm not sure we want to include these styles by default.  I think it's great to enable the ability to override these values in your own app, and it seems like maybe we need the changes to use `AppCompatAlertDialog` that you've made in this PR (thanks!) to facilitate this.

Reply:
I don't see how changing Android theme could override Maui .xaml styles.
Because Maui styles instruct handlers to set properties on concrete UI elements, and these take precedence over theme-defined values in Android.